src/legacy/server/config/schema.js

```diff
@@ -71,19 +71,6 @@ export default () =>
     server: Joi.object({
       name: Joi.string().default(os.hostname()),
```

Comment:
`HANDLED_IN_NEW_PLATFORM` states `This key is handled in the new platform ONLY`, however `server.name` is still used in legacy in some places such as
https://github.com/elastic/kibana/blob/254b18c20b7e90d4f7967faf20625af50591494c/src/legacy/ui/ui_render/ui_render_mixin.js#L212

so I kept it as is. should I change it to `name: HANDLED_IN_NEW_PLATFORM` ?

Reply:
The validation is handled in the new platform, so I'd remove this. The legacy platform can continue using `config.get('server.name')`, but validation will be delegated to the NP.

Reply:
TIL we can't. Some plugins are still accessing it from the legacy config, and as we are constructing the legacy config from the raw configuration values, any default from our NP schemas are ignored. had to keep it for now.